### ISSUE TITLE
`pkgx install pkg` throws error when called twice

### ISSUE THREAD

**Issue by mfts** [pkgxdev/pkgx#796]
accidentally wrote the same command twice and was surprised by that it encountered an error the second time. 

```sh
$ pkgx install gh
pkgx: installed: ~/.local/bin/gh

$ pkgx install gh
× unexpected error file-exists:/Users/mfts/.local/bin/gh
│ Error: file-exists:/Users/mfts/.local/bin/gh
│     at Path.write (https://deno.land/x/libpkgx@v0.14.1/src/utils/Path.ts:344:25)
│     at write (file:///Users/builder/actions-runner/_work/pkgx/pkgx/src/modes/install.ts:20:11)
│     at eventLoopTick (ext:core/01_core.js:183:11)
│     at async default (file:///Users/builder/actions-runner/_work/pkgx/pkgx/src/modes/install.ts:11:5)
│     at async default (file:///Users/builder/actions-runner/_work/pkgx/pkgx/src/app.ts:83:7)
│     at async file:///Users/builder/actions-runner/_work/pkgx/pkgx/entrypoint.ts:59:3
╰─➤ https://github.com/pkgxdev/pkgx/issues/new
```

### expectations

invoking `pkgx install pkg` a second time will say something like 

```sh
pkgx: already installed: ~/.local/bin/gh
```

or just repeat the same message from the first install

**Comment by jhheider** [comment]
geez, who doesn't love an unexpected and complicated error message when they make a minor mistakes?

**Comment by mxcl** [comment]
Already fixed in 1.0.1. Dupes #753
